### PR TITLE
Sorting imports and removing superfluous import

### PR DESCRIPTION
**Pull request by thopiekar**
.. Logger has been imported twice..

### CODE REVIEW

File: plugins/RemovableDriveOutputDevice/RemovableDrivePlugin.py [hunk 1]
```diff
@@ -1,15 +1,19 @@
 # Copyright (c) 2015 Ultimaker B.V.
 # Uranium is released under the terms of the AGPLv3 or higher.
 
+# Python build-ins
```
Comment (fieldOfView) [inline plugins/RemovableDriveOutputDevice/RemovableDrivePlugin.py]: built-ins?

**Comment by thopiekar** [comment]
Well, that's how I call them 🙄
The modules, which normally come with the Python sources including the interpreter itself..

**Comment by fieldOfView** [comment]
build vs built (present vs past perfect tense). I generally don't build Python, but I use builds that someone else built. And he/she built in the built-in modules in that build (probably while sitting in a building).

**Comment by nickthetait** [comment]
Isn't english a lovely language :upside_down_face:

**Comment by Ghostkeeper** [comment]
Still works. It seems that the Message is also an unused import, but I'll see to that myself (and the spelling).

**Pull request merged**